code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -4405,6 +4405,33 @@
 
 	M.nutrition += nutriment_factor
 
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		switch(volume)
+			if(1 to 15)
+				if(prob(5))
+					to_chat(H,"<span class='warning'>You burp.</span>")
+					holder.remove_reagent(src.id, 0.1 * FOOD_METABOLISM)
+			if(15 to 200)
+				if(prob(10))
+					to_chat(H,"<span class='warning'>You really don't feel very good.</span>")
+				if(prob(5))
+					to_chat(H,"<span class='warning'>You feel a burn in your chest.</span>")
+					var/datum/organ/internal/heart/L = H.internal_organs_by_name["heart"]
+					if(istype(L))
+						L.take_damage(0.2, 1)
+			if(100 to INFINITY)//Too much corn oil holy shit, no one should ever get this high
+					if(H.get_heart())//Got a heart?
```

Comment:
Indentation gore

Reply:
what shify said. Also you don't want to do `H.get_heart()` twice